src/helpers/anonymizer.js

```diff
@@ -11,9 +11,25 @@ const configDir = (() => {
 
 const cwd = typeof process === 'object' ? process.cwd() || '.' : '__NOTHING_TO_REPLACE__'
 
+// all the paths the app will use. we replace them to anonymize
+const basePaths = {
+  $USER_DATA: configDir,
+  '.': cwd,
+}
+
 function filepathReplace(path: string) {
-  if (!cwd) return path
-  return path.replace(cwd, '.').replace(configDir, '$USER_DATA')
+  if (!path) return path
```

Comment:
if no path return path ?

Reply:
yeah. that was what was supposed to be the previous code. there is no point replacing an empty string.